### PR TITLE
[chore] move prometheus exporter to generated lifecycle tests

### PR DESCRIPTION
**Pull request by Frapschen**
Relates to https://github.com/open-telemetry/opentelemetry-collector-contrib/issues/27849

### CODE REVIEW

File: exporter/prometheusexporter/metadata.yaml [hunk 1]
```diff
@@ -15,3 +15,8 @@ status:
   - liatrio
   codeowners:
     active: [Aneurysm9]
+
+tests:
+  config:
+    endpoint: "1.2.3.4:1234"
+  skip_lifecycle: true
```
Comment (codeboten) [inline exporter/prometheusexporter/metadata.yaml:22]: is skipping necessary for this exporter? i'd expect that setting the endpoint to `127.0.0.1:<some random port>` would be enough to make this pass no?

Comment (codeboten) [reply exporter/prometheusexporter/metadata.yaml:22]: @Frapschen does this ^^ not work?

**Review by codeboten — COMMENTED** [review]
Pinging code owner @Aneurysm9 please review

File: exporter/prometheusexporter/metadata.yaml (hunk 1 above)
Comment (Frapschen) [reply exporter/prometheusexporter/metadata.yaml]: it works fine

**Review by dmitryax — APPROVED** [review]

**Comment by dmitryax** [comment]
@Frapschen please rebase and take a look at the faining CI

**Review by dmitryax — APPROVED** [review]

**Comment by dmitryax** [comment]
Ci is still failing

**Review by dmitryax — APPROVED** [review]

**Pull request merged**